templates/resources/personalize_dataset_group.md.tmpl

```diff
@@ -0,0 +1,31 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Basic Usage Example (Domain)
+Description about the first example
```

Comment:
can you provide 1 liner about what this example is for, right now it still defaulting to the boilerplate